modules/standard/Map.chpl

```diff
@@ -513,7 +513,7 @@ module Map {
 
     /* Remove the element at position `k` from the map and return its value
      */
-    deprecated "'Map.getAndRemove' is deprecated. Please index into the map and then remove the value."
+    deprecated "'Map.getAndRemove' is deprecated. Consider 'Map.get' and 'Map.remove' instead?"
```

Comment:
Your test of this deprecation message will probably need an update as well, if you haven't run a paratest and seen it :)